app/controllers/provider_interface/content_controller.rb

```diff
@@ -53,5 +53,9 @@ def dates_and_deadlines
     def roadmap
       render_content_page :roadmap
     end
+
+    def organisation_permissions
+      render_content_page :organisation_permissions
```

Comment:
Does this page need breadcrumbs, similar to the dates and deadline one?
<img width="407" alt="image" src="https://user-images.githubusercontent.com/47917431/170437373-f665e963-6c4f-4f74-b104-2b34781198c9.png">